src/containers/auth/Register.js

```diff
@@ -8,6 +8,7 @@ class Register extends Component {
   state = {
     email: '',
     password: '',
+    password_confirmation: '',
```

Comment:
ukoliko prije nije postojao password_confirmation i u specifikaciji nigdje nije napisano da bi trebao da bude, onda to nije tvoj posao da nameces implementaciju registracije. Takve stvari dolaze od klijenta, sutra ces imati svakakve zahtjeve koji i nece imati neku posebnu smisao, a moraces da odradis na nacin na koji je to trazeno, tako da slazem se da bi trebalo ali ukoliko takav zahtjev ne postoji onda ne treba raditi

Reply:
U inicjalnom projektu je za password validaciju bilo stavljeno 'confirmation' pa sam ukapirao da treba da se odradi password_confirmation. Da menjam ili ne?

Reply:
aha, okej onda, ma ne moras, okej je, samo da ti bude ovo sto sam ti napisao u glavi kad sutra budes radio na nekom projektu, da uvijek gledas koji su zahtjevi